example/test/unit_test.dart

```diff
@@ -1,8 +1,8 @@
-import 'package:test/test.dart';
+// import 'package:test/test.dart';
```

Comment:
I think you may remove `unit_test.dart` from commit, if you are not making any update this this.